File: Android Kotlin Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/orphanslife/model/Role.kt
package com.kaligotla.orphanslife.model

import com.google.gson.annotations.SerializedName

data class Role(
    @SerializedName("role_id")
    val role_id: Int,

    @SerializedName("role")
    val role: String,

    @SerializedName("role_created_at")
    val role_created_at: String,

    @SerializedName("role_updated_at")
    val role_updated_at: String,
)

File: MongoDB Express - API - Orphanslife/Data/model/Role.js
const mongoose = require('mongoose')
const Schema = mongoose.Schema
const passportLocalMongoose = require('passport-local-mongoose');

var Role = new Schema({
    role_id:{
        type: Number
    },
    role:{
        type: String
    },
    role_created_at:{
        type: String
    },
    role_updated_at:{
        type: String
    }
}, { collection: 'role' })

Role.plugin(passportLocalMongoose);
  
module.exports = mongoose.model('Role', Role)